product_available_by_warehouse/models/sale.py

```diff
@@ -62,7 +62,8 @@ def onchange_show_message(self, show_message, product_id):
         # update route domain
         if route_ids:
             domain = res.get('domain', {})
-            domain['route_id'] = [('id', 'in', route_ids)]
+            # TODO: uncomment until figure out a better way
+            # domain['route_id'] = [('id', 'in', route_ids)]
             res.update({'domain': domain})
```

Comment:
@nhomar 

this code do not passing blank domain, in fact is passing the exactly same domain returned by the original onchange

this change is because before it was modifying in wrong way and that is the reason of the change and  `TODO`